packages/cardano-services/test/NetworkInfo/NetworkInfoHttpService.test.ts

```diff
@@ -58,6 +83,10 @@ describe('NetworkInfoHttpService', () => {
     });
 
     describe('/network', () => {
+      const path = '/network';
+      const dbSyncPollQueriesCount = 1;
```

Comment:
```suggestion
      const DB_POLL_QUERIES_COUNT = 1;
```
1. We've been using this format for `number`, `bigint`, or `string` constants, although it would be good to have a lint rule to enforce it, so I'm ok with it either way.
2. We can just refer to this as DB polling since we're interfacing with PostgreSQL. `cardano-db-sync` is the projector.

Reply:
[6e5ee7c](https://github.com/input-output-hk/cardano-js-sdk/pull/276/commits/6e5ee7c71babfe32d15e36f92ea25177a42f75a5)